Makefile

```diff
@@ -71,7 +71,19 @@ source-pypi: clean
                 fi;\
 	done
 
-pypi: source-pypi develop
+wheel: clean
+	if test ! -d PYPI_UPLOAD; then mkdir PYPI_UPLOAD; fi
+	$(PYTHON) setup.py bdist_wheel -d PYPI_UPLOAD
+	for PLUGIN in $(AVOCADO_OPTIONAL_PLUGINS); do\
+		if test -f $$PLUGIN/setup.py; then\
+			echo ">> Creating wheel distribution for $$PLUGIN";\
+			cd $$PLUGIN;\
+			$(PYTHON) setup.py bdist_wheel -d ../../PYPI_UPLOAD;\
+			cd -;\
+                fi;\
+	done
+
+pypi: wheel source-pypi develop
```

Comment:
cool! I'm assuming, since you already uploaded the wheel packages, that's the verification we need.